docs/dev_tools/build_profiles.md

```diff
@@ -0,0 +1,21 @@
+# Build Profiles
+mbed 5.0 supports three primary build profiles, *default*, *debug* and *small*. The Online 
+Compiler uses the *default* profile. When building from the command-line,
```

Comment:
This should explicitly name mbed CLI, not just the general idea of a command-line